dcpy/builds/load.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+import typer
+
+from dcpy import REPO_ROOT_PATH
+from dcpy.utils import git, postgres
+from dcpy.utils.logging import logger
+from dcpy.connectors.edm import recipes
+
+
+def setup_environment(pg_client: postgres.PostgresClient):
+    pg_client.drop_schema()
+    pg_client.create_schema()
+    # TODO delete draft S3 folder
+
+
+def load_source_data(product: str):
+    product_path = REPO_ROOT_PATH / f"products/{product}"
```

Comment:
I may be wrong here, but I suspect, unfortunately, we'll need to supply the path to the product. Since `dcpy` is installed via pip, the `REPO_ROOT_PATH` might have some surprising results when `dcpy` isn't installed in editable mode. Looks like the only other usage of that constant is to source an env var... it may be time to kill it.

Reply:
Hmm. A build should be a decent test of this, no? We're doing a standard pip install at the beginning of build steps

Reply:
@fvankrieken it might, so long as you're invoking python from outside our project root, otherwise I _think_ the python interpreter will probably just use the existing dcpy folder. (unless we're deleting that somewhere?)

Testing this locally, and if I replicate the container setup step install locally with just (in proj root)
`python3 -m pip install . -c ./python/constraints.txt`

then cd into, say, the projects folder, then invoke python, I see that 
REPO_ROOT_PATH=/workspace/.venv/lib/python3.11/

if I run python from workspace root, it ignores the installed version (which makes sense) and 
REPO_ROOT_PATH=/workspace/

Reply:
Ah makes sense. 

These are intended to be run in the product build folders, right? So maybe a supplied arg for path to folder, with default to `"."` (no repo root path needed)?

Reply:
@fvankrieken I think one of the strengths of installing dcpy via pip is that we aren't tied to running from proj root anymore. I could also see a world in which other teams (e.g. GIS) add `dcpy` as a dependency. So I think we have no guarantees about where `dcpy` lives on a filesystem. But as Tim from AE says, that's a tomorrow issue.

For this, seems most straightforward to expect a path.

Reply:
Oh, yes, I see. Totally sensible!